src/components/BRB/BRBMainComponent.tsx

```diff
@@ -505,9 +505,11 @@ const ButtonItem = styled(Button)`
   vertical-align: middle;
   font-size: 18px;
   font-style: normal;
+  font-family: Glancyr;
```

Comment:
fallback fonts should be mentioned: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family